src/components/reimbursement-table/column-def.tsx

```diff
@@ -0,0 +1,79 @@
+import { formatAmount, formatDate } from "@/utils/format";
+import { ColumnDef } from "@tanstack/react-table";
+import { Badge } from "../ui/badge";
+import { Button } from "@/components/ui/button";
+import { DownloadIcon } from "@radix-ui/react-icons";
+
+export type Reimbursement = {
```

Comment:
when the reimbursement schema PR gets merged, could we change all references to refer to the schema type instead of this for consistency? for now, this works great